troposphere/ecs.py

```diff
@@ -26,6 +26,20 @@ class DeploymentConfiguration(AWSProperty):
     }
 
 
+class PlacementConstraint(AWSProperty):
+    props = {
+        'Type': (basestring, True),
+        'Expression': (basestring, False),
+    }
+
+
+class PlacementStrategy(AWSProperty):
+    props = {
+        'Type': (basestring, True),
```

Comment:
Quite a bit of the validation was put in later in the project and there has never been an effort to go back to add it. If there is an easy way to constrain the value it is always good to add it now. This can be done like the emr examples of using a function instead of basestring, or easier by adding a validate() method to the object. The former will validate at object creation time and the latter at template creation time. I'm fine with either since that is better than having stacks rollback with illegal options.